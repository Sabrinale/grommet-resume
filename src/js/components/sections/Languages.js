import React from 'react'

import { SectionTitle } from '../SectionTitle'
import { Highlights } from '../Highlights'

const Language = ({ item }) => (
  <Highlights>
    <SectionTitle>{`${item.language}: ${item.fluency}`}</SectionTitle>
  </Highlights>
)

export { Language }
