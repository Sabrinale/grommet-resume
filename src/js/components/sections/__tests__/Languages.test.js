import React from 'react'
import renderer from 'react-test-renderer'

import { Language } from '../Languages'

import defaultResume from './mockResume.json'

test('renders languages section', () => {
  const data = defaultResume.languages[0]
  const component = renderer.create(<Language item={data} />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
