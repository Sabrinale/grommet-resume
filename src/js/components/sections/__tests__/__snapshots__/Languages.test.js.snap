// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders languages section 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 kFdcfG"
>
  <h3
    className="StyledHeading-sc-1rdh4aw-0 jVEgcZ"
  >
    English: Native speaker
  </h3>
</div>
`;
